bgp/modules/terms.py

```diff
@@ -6,16 +6,33 @@
 from lxml import etree
 
 
-STOP_WORDS = set("""a about above after again against all almost also am among an and
-    any are around as at be became because been before being below between both but by
-    can could did do does doing don down during each ever far few for from further had
-    has have having he her here hers herself him himself his how i if in into is it its
-    itself just like many may me more most much my myself new no nor not now of off
-    often on once one only or other our ours ourselves out over own s same see seems
-    she should so some such t than that the their theirs them themselves then there
-    these they things this those three through to too two under until up very was we
-    well were what when where which while who whom why will with would you your yours
-    yourself yourselves""".split())
+STOP_WORDS = {
```

Comment:
let's make it a list using `[]`  rather than a dictionary

Reply:
Why are we changing this from a set to a set?...

Reply:
@tabshaikh also, this is not a dictionary. It's a set. And the purpose is that a set only stores elements once and doesn't require an order. It's more efficient for this use case.

Reply:
@mekarpeles 
We are adding more stop words that were not present in the previous set for eg. .
These stop words are taken from an NLP library (spacy) itself view [this ](https://github.com/explosion/spaCy/blob/master/spacy/lang/en/stop_words.py) for more details.
Directly using the pre-processed STOP_WORDS result saves us time and improves the efficiency of our algorithm.

Reply:
> @cclauss originally switched it from a set to a string split and I think we should keep his change as it's easier to update.
@mekarpeles 
So should I close this PR or should I update the PR with the same method as @cclauss ?
Having more set of stop words will ultimately help us to clean up the irrelevant words when we process a large amount of text.

Reply:
My vote would be to add more words in the multi line form.

It is fast and easy to read (as long as new entries preserve the sort order).